File: 3842-toggle-light-bulbs/3842-toggle-light-bulbs.java
class Solution {
    public List<Integer> toggleLightBulbs(List<Integer> bulbs) {
        HashSet<Integer> set=new HashSet<>();
       for(int n :  bulbs){
           if(set.contains(n)){
               set.remove(n);
           }else{
           set.add(n);
           }
       }
        List<Integer> res=new ArrayList<>(set);
        Collections.sort(res);
        return res;
        
    }
}

File: 3842-toggle-light-bulbs/3842-toggle-light-bulbs.cpp
class Solution {
public:
    vector<int> toggleLightBulbs(vector<int>& bulbs) {
      map<int,int>freq;
      vector<int>result;
      for(int i=0;i<bulbs.size();i++){
        freq[bulbs[i]]++;
      }
      for(auto it:freq){
        if(it.second%2!=0){
            result.push_back(it.first);

        }
      }
      return result;
    }
};